tests/integration/kubernetes/tests_common.sh

```diff
@@ -103,6 +103,8 @@ adapt_common_policy_settings_for_cbl_mariner() {
 	info "Adapting common policy settings for CBL-Mariner"
 	jq '.request_defaults.UpdateEphemeralMountsRequest = true' "${settings_dir}/genpolicy-settings.json" > temp.json && sudo mv temp.json "${settings_dir}/genpolicy-settings.json"
 	jq '.kata_config.oci_version = "1.1.0-rc.1"' "${settings_dir}/genpolicy-settings.json" > temp.json && sudo mv temp.json "${settings_dir}/genpolicy-settings.json"
+	jq '.sandbox.storages += [{"driver":"virtio-fs","driver_options":[],"fs_group":null,"fstype":"virtiofs","mount_point":"/run/kata-containers/shared/containers/","options":[],"source":"kataShared"}]' \
```

Comment:
Wait, why is this being added? :-)

Reply:
This was added to fix the tests on mariner. Otherwise we get [blocked](https://github.com/kata-containers/kata-containers/actions/runs/11023710813/job/30657850953?pr=10340#step:13:424)

Reply:
I see, for some reason I thought mariner was virtiofs-free.

Let's get this one in, as we know for sure it doesn't cause regressions, and we can discuss later how to get those tested, as it's not going to be the easiest one.